app/account/forms.py

```diff
@@ -37,6 +37,12 @@ class RegistrationForm(Form):
         Length(1, 64),
         Email()
     ])
+    phone_number = TelField('Phone Number', validators=[
+        Length(0, 15),
+        Regexp(r'^[0-9]*$',
```

Comment:
We discussed this a little during meeting, I think we said we'd consider allowing symbols but removing them later?